osgi/integrationtest/src/test/java/mockservice/MockService.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Hibernate Validator, declare and validate application constraints
+ *
+ * License: Apache License, Version 2.0
+ * See the license.txt file in the root directory or <http://www.apache.org/licenses/LICENSE-2.0>.
+ */
+
+package mockservice;
+
+import com.example.CustomerDecimalMin;
+import org.hibernate.validator.HibernateValidator;
+import org.hibernate.validator.HibernateValidatorConfiguration;
+import org.hibernate.validator.messageinterpolation.ResourceBundleMessageInterpolator;
+import org.hibernate.validator.resourceloading.PlatformResourceBundleLocator;
+
+import javax.el.ExpressionFactory;
+import javax.validation.ConstraintViolation;
+import javax.validation.Validation;
+import javax.validation.ValidationProviderResolver;
+import javax.validation.Validator;
+import javax.validation.spi.ValidationProvider;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+
+/**
+ *
+ */
+public final class MockService {
+    private MockService() {
+        //
+    }
+
+    public static Set<ConstraintViolation<CustomerDecimalMin>> run( ClassLoader testClassLoader ) {
+        CustomerDecimalMin customer = new CustomerDecimalMin();
+
+        HibernateValidatorConfiguration configuration = Validation.byProvider( HibernateValidator.class )
+                        .providerResolver( new ValidationProviderResolver() {
+                            @Override
+                            public List<ValidationProvider<?>> getValidationProviders() {
+                                ValidationProvider<HibernateValidatorConfiguration> prov = new HibernateValidator();
+                                List<ValidationProvider<?>> provs = new ArrayList<>();
+                                provs.add( prov );
+                                return provs;
+                            }
+                        }).configure();
+
+        ClassLoader oldTccl = Thread.currentThread().getContextClassLoader();
+        ExpressionFactory expressionFactory;
+        try {
+            // use _our_ classloader, which has com.sun.el and all that, to obtain
+            // an expression factory.
+            Thread.currentThread().setContextClassLoader( MockService.class.getClassLoader() );
```

Comment:
That should not be needed any longer? I think overall it'd be nice to just amend the existing test case to cover `@DecimalMin` (and thus the EL stuff), too.